gazebo_ros2_control/CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ ament_target_dependencies(gazebo_hardware_plugins
   gazebo_dev
   hardware_interface
   rclcpp
+  urdf
```

Comment:
A better approach is to remove this header in these two files:

 - gazebo_ros2_control/include/gazebo_ros2_control/gazebo_system_interface.hpp
 - gazebo_ros2_control/src/gazebo_ros2_control_plugin.cp

Reply:
Done in [0c290bc](https://github.com/ros-simulation/gazebo_ros2_control/pull/117/commits/0c290bcb9570e47d3b3d253a0df2261136877348)